lisa/base_tools/cat.py

```diff
@@ -11,3 +11,12 @@ def command(self) -> str:
 
     def _check_exists(self) -> bool:
         return True
+
+    def read_from_file(
+        self,
+        file_path: str,
+        force_run: bool = False,
+        sudo: bool = False,
+    ) -> str:
+        # Run `cat <file_path>`
+        return self.run(file_path, force_run=force_run, sudo=sudo).stdout
```

Comment:
raise exception, if the file doesn't exist. It can be implemented by assert the exit code.